railgun/internal/ctrlutils/ingress-status.go

```diff
@@ -87,8 +87,10 @@ func UpdateIngress(ctx context.Context, targetContent *file.Content, log logr.Lo
 					for _, header := range config.(map[string]interface{})["headers"].([]interface{}) {
 						if strings.HasPrefix(header.(string), "Knative-Serving-") {
 							log.Info("knative service updated. update knative CR condition and status...")
-							err := UpdateKnativeIngress(ctx, log, svc, kubeConfig, ips, hostname)
-							return fmt.Errorf("failed to update knative ingress err %v", err)
+							if err := UpdateKnativeIngress(ctx, log, svc, kubeConfig, ips, hostname); err != nil {
+								return fmt.Errorf("failed to update knative ingress err %v", err)
```

Comment:
The following is more consistent with our error wrapping throughout the codebase and allows the caller to `Unwrap` if they so choose:

```suggestion
								return fmt.Errorf("failed to update knative ingress: %w", err)
```

Reply:
Kindly see below comments and leave as it is atm.